portfolio/pom.xml

```diff
@@ -12,11 +12,11 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>11.0.0</jetty.version>
+    <jetty.version>9.4.31.v20200723</jetty.version>
```

Comment:
This PR seems to include changes from a prior PR. Please merge prior PR and rebase with master to exclude these changes.